qlib/rl/order_execution/policy.py

```diff
@@ -163,6 +163,9 @@ def auto_device(module: nn.Module) -> torch.device:
 def load_weight(policy: nn.Module, path: Path) -> None:
     assert isinstance(policy, nn.Module), "Policy has to be an nn.Module to load weight."
     loaded_weight = torch.load(path, map_location="cpu")
+
+    if "vessel" in loaded_weight:
```

Comment:
I think this should be handled by whoever calls `load_weight`.
For example, when the outer class receives a weight, it should first unpack it, and send the corresponding part to individual component.

This `if` can work as a hack though. Please leave a TODO here in that case.